helm/harbor-operator-app/crd/crd-chartmuseums.yaml

```diff
@@ -0,0 +1,4337 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: 'harbor-operator/harbor-operator-serving-cert'
```

Comment:
The crds were previously templated using the `{{ Release.Namespace }}/ {{ . | include "chart.fullname" }}`. Since the `crd-install` mechanism doesn't template files prior to installation these values have been hard-coded. The namespace aligns with the harbor-operator namespace set within the cluster-management-fleet open PR for harbor deployment.